vendor/assets/javascripts/jquery.minicolors.simple_form.js

```diff
@@ -1,4 +1,4 @@
-$(function() {
+$(document).on('ready turbolinks:load', function {
```

Comment:
In the mean time, I removed the `//= require jquery.minicolors.simple_form` line from **application.js** and add this to the end of that file. It works correctly. So, just add the parenthesis and it's good to go. :tada: